# -*- coding: utf-8 -*-
from pylibs import PyLibsException


class CallTrackingException(PyLibsException):
    pass
